src/requirements.txt

```diff
@@ -1,21 +1,16 @@
-django~=3.2.16
-djangorestframework~=3.11.0
-django-rest-swagger~=2.2.0
-drf-yasg~=1.17.0
-mozilla-django-oidc~=1.2.3
-whitenoise~=5.0.1
-dj-database-url~=0.5.0
-requests~=2.23.0
-django-watchman~=1.1.0
-django-webpack-loader~=0.6.0
-django-safedelete~=0.5.4
-jsonfield~=3.1.0
+django==3.2.18
+djangorestframework==3.14.0
```

Comment:
You'll notice I changed all these dependencies to hard pins from allowing patch versions. I can be convinced to go back, but it seems preferable to have reproducible builds and avoid possible bugs in these patches. Most of T&L's projects use pins.

Reply:
This is probably better now with dependabot watching for these. It's not going to be able to tell as well if we need an upgrade with the ~= versions.